frontend/src/console/components/NoticeBoard/displayMessages.ts

```diff
@@ -25,8 +25,24 @@ const messages = {
     defaultMessage: 'Deployment offline.',
     description: 'Redeployment alert message.'
   },
+  explanation: {
+    defaultMessage:
+      'There are many factors that affect the speed at which updates will ' +
+      'reflect on your site. Please allow up to 2 hours for your site to be updated.',
+    description: 'Brief explanation on deployment duration.'
+  },
+  blogpost_text: {
+    defaultMessage:
+      "If you'd like to read more about the way we provision the sites, " +
+      'you can read our blog post about the process ',
```

Comment:
Was this text on the designs? Wouldn't it sound better like this?
```js
"If you would like to learn more about the way we provision the sites, please see our <link>blog post</link> explaining the deployment process."
```

Reply:
It's not, but it didn't mention the blogpost either. I like your suggestion. Added as such.